Makefile

```diff
@@ -2,6 +2,7 @@ DEBUG = no	# Set to yes, for a debug build.
 BENCHMARK = no	# Set to yes, for an optimized build with debug output.
 THREADED = yes	# Set to no, to disable multithreading.
 PLOT = no	# Set to yes, to enable dumping of filter response plots.
+LIB32= no # Set to yes, to build lib32 version for 64-bit systems
```

Comment:
I was going to suggest adding a space after `LIB32`, for consistent formatting, but I then noticed that this won't work at all.  Bash considers all whitespace after the equal sign as part of the variable's value, so none of the above are ever considered equal to `yes`.  I would therefore suggest changing this to:

```suggestion
# Set to yes, for an optimized build with debug output.
BENCHMARK=no

# Set to no, to disable multithreading.
THREADED=yes

# Set to yes, to enable dumping of filter response plots.
PLOT=no

# Set to yes, to build lib32 version for 64-bit systems
LIB32=no
```

The change should probably be made as a separate commit, for clarity.